doc/manual.rst

```diff
@@ -1502,53 +1502,35 @@ Hyphenation
 
 If you want good looking documents, you want to enable hyphenation.
 
-To do it, you need to install Wordaxe [#]_.
+To do it, you first need to install the ``pyphen`` python module.
 
-.. [#] Use Roberto Alsina's fork of Wordaxe from https://github.com/ralsina/wordaxe as
-       this works with later versions of ReportLab.
+Then, you need to specify the language in each style that you want hyphenation
+to work. To have hyphenation in the whole document, you can do it in the
+``base`` style.
 
-If after installing it you get the letter "s" or a black square instead of a
-hyphen, that means you need to replace the rl_codecs.py file from reportlab with
-the one from wordaxe.
+For example, for an english document, hyphenation can be turned on for the whole
+document with::
 
-For more information, see `this issue`_ in rst2pdf's bug tracker.
-
-.. _this issue: http://code.google.com/p/rst2pdf/issues/detail?id=5
-
-Also, you may need to set hyphenation to true in one or more styles, and the
-language for hyphenation via the command line or paragraph styles.
-
-For English, this should be enough::
-
-    ["bodytext" , {
-      "alignment": "justify",
-      "hyphenation": true
+    ["base" , {
+      "hyphenationLang": "en_US",
+      "embeddedHyphenation": 1
     }],
 
-If you are not an English speaker, you need to change the language using
-the ``-l`` or ``--language`` option.
-
-Since Wordaxe version 0.2.6, it can use the PyHyphen library if it's
-available. PyHyphen can use any OpenOffice dictionary, and can even download
-them automatically. [#]_
-
-.. [#] For more information, please check the PyHyphen website at http://pyhyphen.googlecode.com
-
-For example, this will enable German hyphenation globally::
-
-    rst2pdf -l de_DE mydocument.txt
+Notice the ``embeddedHyphenation`` option. It is optional, but it makes so that
+hyphenations will give preference to splitting words in embedded hyphens in the
```

Comment:
Please change the _in_ in "words in embedded hyphens" to _with_, so that it reads "words with embedded hyphens".

Reply:
I guess what I meant was actually `at` rather than `in`. So, "splitting words at embedded hyphens". Because from my understanding that option tries to split the word at the embedded hyphen if it is present.
Which do you think is the best?

Reply:
`at` works better for me.